.travis.yml

```diff
@@ -1,12 +1,30 @@
 language: php
+
+sudo: false
+
 php:
   - 5.5
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
@jpignata 

As pointed out above, when builds are sent to container-based infrastructure, caching can be used.

The change above allows to cache dependencies installed with `composer` between builds, meaning that builds run faster, as dependencies do not need to be fetched from remote (unless they have changed, of course).

See [Arbitrary directories](https://docs.travis-ci.com/user/caching#Arbitrary-directories).